public/index.html

```diff
@@ -0,0 +1,22 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <title>Discovery Demo</title>
+    <meta charSet="utf-8" />
+    <meta httpEquiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="og:title" content="Discovery Demo" />
+    <meta name="og:description" content="%REACT_APP_DESCRIPTION%" />
+    <link rel="icon" type="image/x-icon" href="%PUBLIC_URL%/static/images/favicon.ico">
+    <!-- Begin Bluemix Analytics -->
+    <meta name="segment" property="watson-demos" value="discovery-news-demo" />
+    <script type="text/javascript">
+      window._analytics = { coremetrics: false, optimizely: false, addRoll: false };
+    </script>
+    <script type="text/javascript" src="%REACT_APP_BLUEMIX_ANALYICS%"></script>
+    <!-- End Bluemix Analytics -->
+  </head>
+  <body>
+    <div id="root"></div>
+  </body>
+</html>
```

Comment:
so create react app uses this file to interpolate variables for static assets (like the google analytics and favicon) then injects all the react junk into the div with the `id="root"`